catalog/nexus/nexus.bom

```diff
@@ -0,0 +1,88 @@
+brooklyn.catalog:
+  version: &nexusVersion "1.0.0-SNAPSHOT" # BROOKLYN_NEXUS_VERSION
+  iconUrl: &nexusIconUrl "https://github.com/sonatype.png"
+  name: &nexusName "Nexus Repository OSS"
```

Comment:
Would be great to directly include the [`publish`](https://github.com/brooklyncentral/blueprint-repository) block to release on blueprint.cloudsoft.io